docs/csharp/cs-dev-kit-faq.md

```diff
@@ -71,6 +71,16 @@ Note that, depending on your network speed, installing the .NET Core runtime mig
 }
 ```
 
+### Error Acquiring .NET 7
+
+![Error Acquiring .NET 7](images/faq/error-acquiring-dotnet-7.png)
+
+Note: If you are located in China the Great Firewall of China may be blocking your .NET download which can cause a timeout.
```

Comment:
Are we allowed to say "Great Firewall of China"?
Might be better to say "If you are located in China, your .NET download may be blocked and cause a timeout."